### DISCUSSION TITLE
Div component to use css prop

### DISCUSSION THREAD

**Discussion by buster95** [heroui-inc/heroui#734] [Feature requests]
Is there any way to create a custom component using css property??

I would like to avoid this
![image](https://user-images.githubusercontent.com/15637669/188303265-bb4ccd37-289a-451a-b656-703b359b97fb.png)

**Comment by jrgarciadev** [comment]
Hey this has nothing to do with NextUI

**Comment by buster95** [comment]
Is there a way to create a component with your custom styles?? in @mui there are some components to use custom style for example <Box />
